src/Microsoft.TemplateEngine.Authoring.CLI/LocalizableStrings.resx

```diff
@@ -182,6 +185,9 @@ Do not localize: "template.json"</comment>
     <value>If set to true - the diff tool won't be automatically started by the Verifier on verification failures.</value>
     <comment>Do not translate 'true'.</comment>
   </data>
+  <data name="command_verify_help_distinguisherName_description" xml:space="preserve">
+    <value>Sets optional distinguisher string prepended to the generated scenario name used for naming expectations subdirectories</value>
```

Comment:
nit:
```suggestion
    <value>Specifies scenario name to be used in the snapshot folder name.</value>
```

I'm not sure if `distinguisher` is correct term :), for me it only associates with
![image](https://user-images.githubusercontent.com/56045248/196636957-306effea-95d3-44f1-bd80-c4a51673f65d.png)